nagiosplugin/output.py

```diff
@@ -39,7 +39,7 @@ def format_status(self, check):
         summary_str = check.summary_str.strip()
         return self._screen_chars('{0}{1}{2}'.format(
             name_prefix, str(check.state).upper(),
-            ' - ' + summary_str if summary_str else ''), 'status line')
+            ' - ' + summary_str if summary_str else ''), 'status line').strip()
```

Comment:
Is this directly related to the change in `check.py`?  It looks like an unrelated modification.